### PR TITLE
Or 1624 merge and pr uniswap v3 predeploy logics functionality check done into ecotone version

### PR DESCRIPTION
**Pull request by AaronLee22**
### This is all the codes for predeploy UniswapV3  contracts.

- Updated all the immutable constructors variables/storage variables
- Updated Hardartifacts(predeploy UniswapV3)
- Updated bindings(predeploy UniswapV3) hardhat version
- Updated immutableReferences (T/F) hardhat version

### CODE REVIEW

**Comment by rlgns98kr** [comment]
@AaronLee22 
I just have re-run action, It has been success. I don't know why....
I think we can see it later! 🙇🏼‍♂️

File: packages/tokamak/contracts-bedrock/hardhat.config.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      LOWEST_OPTIMIZER_COMPILER_SETTINGS,
+    ],
+  },
+  networks: {
+    devnetL1: {
+      url: "http://localhost:9545",
+      chainId: 901,
+    },
+    'thanos-sepolia-test': {
+      url: "https://explorer.thanos-sepolia-test.tokamak.network",
```
Comment (rlgns98kr) [inline packages/tokamak/contracts-bedrock/hardhat.config.ts:32]: -> https://rpc.thanos-sepolia-test.tokamak.network 🙇🏼‍♂️😍

File: packages/tokamak/contracts-bedrock/hardhat.config.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        urls: {
+          apiURL: "http://localhost/api",
+          browserURL: "http://localhost",
+        },
+      },
+      {
+        network: "thanos-sepolia-test",
+        chainId: 111551118080,
+        urls: {
+          apiURL: "https://rpc.titan-sepolia.tokamak.network",
```
Comment (rlgns98kr) [inline packages/tokamak/contracts-bedrock/hardhat.config.ts:51]: ->
https://explorer.thanos-sepolia-test.tokamak.network/api

File: packages/tokamak/contracts-bedrock/hardhat.config.ts (hunk 1 above)
Comment (AaronLee22) [reply packages/tokamak/contracts-bedrock/hardhat.config.ts:32]: Thank you for your correction!!

File: packages/tokamak/contracts-bedrock/hardhat.config.ts (hunk 2 above)
Comment (AaronLee22) [reply packages/tokamak/contracts-bedrock/hardhat.config.ts:51]: Thank you for your correction!!

**Comment by AaronLee22** [comment]
> @AaronLee22 I just have re-run action, It has been success. I don't know why.... I think we can see it later! 🙇🏼‍♂️

I see thank you for your help!!!

**Comment by AaronLee22** [comment]
> @AaronLee22 I just have re-run action, It has been success. I don't know why.... I think we can see it later! 🙇🏼‍♂️

I went over the codes but I'm not sure either, thanks for your help Steven!

**Review by rlgns98kr — APPROVED** [review]
Thank you very much! 🥰🙇🏻‍♂️

**Pull request merged**